pyaedt/modeler/geometry_operators.py

```diff
@@ -1551,7 +1551,7 @@ def v_angle_sign_2D(va, vb, right_handed=True):
 
     @staticmethod
     @pyaedt_function_handler()
-    def point_in_polygon(point, polygon):
+    def point_in_polygon(point, polygon, tolerance=1e-8):
         """Determine if a point is inside or outside a polygon, both located on the same plane.
```

Comment:
Can we better explain this part: "both located on the same plane." Both what? Are we referring to the pain and polygon?